.env.example

```
@@ -9,5 +9,5 @@ REDIS_PASSWORD=changeme
 # JWT
 JWT_SECRET=your-256-bit-secret-key-here
 
-# Gemini AI
-GEMINI_API_KEY=your-gemini-api-key
\ No newline at end of file
+# OpenAI API
+OPENAI_API_KEY=your-openai-api-key
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)